server/controllers/gh.js

```diff
@@ -1,17 +1,26 @@
-var url = require('url'),
-    got = require('gh-got'),
-    config = require('../config'),
-    logger = require('../logger'),
-
-    moment = require('moment'),
-    marked = require('marked'),
-
-    Render = require('../render'),
-    render = Render.render,
-    dropCache = Render.dropCache, // eslint-disable-line no-unused-vars
-    requestUrl = [config.ghAPI, 'repos', config.org, config.repo].join('/'),
-    issuesRequestUrl = [requestUrl, 'issues'].join('/'),
-    labelsRequestUrl = [requestUrl, 'labels'].join('/');
+const url = require('url');
+const got = require('gh-got');
+const config = require('../config');
+const logger = require('../logger');
+
+const moment = require('moment');
+const marked = require('marked');
+
+const Render = require('../render');
+const render = Render.render;
+const dropCache = Render.dropCache; // eslint-disable-line no-unused-vars
+const requestUrl = [config.ghAPI, 'repos', config.org, config.repo].join('/');
+const issuesRequestUrl = [requestUrl, 'issues'].join('/');
+const labelsRequestUrl = [requestUrl, 'labels'].join('/');
+
+function withToken(user, body = {}) {
```

Comment:
let's create external wrapper for `got` like this https://nda.ya.ru/3T89kB instead